packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/errorList.markup.tests.js

```diff
@@ -0,0 +1,108 @@
+import $ from 'jquery';
+
+import ErrorList from '__internal/ui/chat/errorlist';
+
+const CHAT_ERRORLIST_CLASS = 'dx-chat-errorlist';
+const CHAT_ERRORLIST_ERROR_CLASS = 'dx-chat-errorlist-error';
+const CHAT_ERRORLIST_ERROR_ICON_CLASS = 'dx-chat-errorlist-error-icon';
+const CHAT_ERRORLIST_ERROR_TEXT_CLASS = 'dx-chat-errorlist-error-text';
+
+const moduleConfig = {
+    beforeEach: function() {
+        const markup = '<div id="errorList"></div>';
+        $('#qunit-fixture').html(markup);
```

Comment:
Isn't it better to do it in `testStart`, not `beforeEach`?

Reply:
There are several reasons why I chose this approach. The main reason is that we use different IDs for each part of the chat and run these parts in single files: chat.tests.js and chat.markup.tests.js. If we use QUnit.testStart(), it will run before the tests start.

While we could use a single ID for all tests or try another solution, that would require refactoring all the test files. I will create a separate card for this and let’s discuss it in the context of separate changes, not in this PR.

Reply:
card has been created. Let's discuss it separately in the next meeting.

Reply:
I prepared PR https://github.com/DevExpress/DevExtreme/pull/28199. We can merge it first to refactor current solution for `errorlist.tests.js` and `errorlist.markup.tests.js`